formatting_scripts/fix_formatting.sh

```diff
@@ -79,7 +79,7 @@ function run_code_spell(){
     fi
 
     printf "Fixing spelling...\n\n"
-    cd $CURR_DIR/../src/software && codespell -w --skip="1,2,0" -D /tmp/dictionary.txt # Skip binaries
+    cd $CURR_DIR/../src/software && codespell -w --ignore-words=codespell_allowed_words.txt --skip="1,2,0" -D /tmp/dictionary.txt # Skip binaries
```

Comment:
this doesn't seem to work for me locally

when you cd to software, the `codespell_allowed_words.txt` file is no longer in the right dir

you might need to add a `../../formatting_scripts`

Reply:
turns out the codespell program we use is a different codespell from the one we get our dictionary from, so `--ignore-words` was never a valid option. Fixed it now by removing the line containing `atleast` from the dictionary